core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala

```diff
@@ -373,7 +372,7 @@ class FunctionPullingContainerProxy(
     case Event(initializedData: InitializedData, _) =>
       context.parent ! Initialized(initializedData)
       initializedData.clientProxy ! RequestActivation()
-      setTimer(UnusedTimeoutName, StateTimeout, unusedTimeout)
+      startTimerAtFixedRate(UnusedTimeoutName, StateTimeout, unusedTimeout)
```

Comment:
hm..i submited another pr to solve this: https://github.com/apache/openwhisk/pull/5123

Reply:
Thanks for review :)  I have updated it.